test/op_shape_test.cpp

```diff
@@ -1396,6 +1396,19 @@ void test_reduce_ops()
         migraphx::shape input{migraphx::shape::float_type, {2, 3, 4, 5}};
         throws_shape(T{{4}}, input);
     }
+    // dynamic shape
+    {
+        migraphx::shape input{migraphx::shape::float_type, {{2, 3, 4}, {2, 4, 4}}};
```

Comment:
This is an odd dynamic dimension. The opt is not even in the range of dynamic values. You should do `{2, 3, 2}` or `{2, 3, 3}`.